exercises/grains/runtests.jl

```diff
@@ -3,10 +3,14 @@ using Test
 include("grains.jl")
```

Comment:
```suggestion
# canonical data version: 1.2.0

include("grains.jl")
```

This isn't documented and not consistent among all test suites yet, but adding a version comment makes updating easier later on.